eng/doc/fips/UserGuide.md

```diff
@@ -859,7 +867,50 @@ The hash.Hash methods are implemented as follows:
 
 ### [crypto/md5](https://pkg.go.dev/crypto/md5)
 
-Not implemented by any backend.
+Package md5 implements the MD5 hash algorithm as defined in RFC 1321.

```

Comment:
Add a note similar to the one about sha1 (but not approved at all for any security purposes)?

> SHA-1 is an approved FIPS 140-2 hash algorithm although it is cryptographically broken and should not be used for secure applications.